.github/workflows/build.yml

```diff
@@ -13,23 +13,23 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [ '7.4', '8.0', '8.1' ]
+        php: [ '8.0', '8.1' ]
```

Comment:
This doesn't include PHP 8.2 because PHP CS FIXER doesn't have a new release that support PHP 8.2 yet.

Do we need to check coding standards for every PHP versions?

How about only run it on latest supported PHP version only? In this case only PHP 8.1?